cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -114,6 +114,28 @@ def test_snapshot_crud(small_test_vm, provider):
     snapshot.delete()
 
 
+@pytest.mark.rhv3
+@pytest.mark.provider([RHEVMProvider], override=True)
```

Comment:
As we are getting `RHEVM` from provider parametrization. I would just uncollect for `VMwareProvider` than override. what do you think?

Reply:
We prefer direct marking, not uncollection. Its the intended use of `override`, and is a much cleaner implementation than having to uncollect with lambda expressions.

Reply:
@mshriver thanks, I didn't know what is the better way